IPs/RSKIP326.md

```diff
@@ -1,56 +1,100 @@
-# Change btcDestinationAddress format in `release_request_received` event
+# Change btcDestinationAddress format to string in `release_request_received` event, create new `pegout_confirmed` event, emit `add_signature` when a tx is actually signed.
 
 |RSKIP          |326           |
 | :------------ |:-------------|
-|**Title**      |Change btcDestinationAddress format in `release_request_received` event |
+|**Title**      |Change btcDestinationAddress format to string in `release_request_received` event, create new `pegout_confirmed` event, emit `add_signature` when a tx is actually signed. |
 |**Created**    |21-JUNE-22 |
-|**Author**     |KI |
+|**Author**     |KI & JT |
 |**Purpose**    |Usa |
 |**Layer**      |Core |
-|**Complexity** |1 |
+|**Complexity** |2 |
 |**Status**     |Draft |
 
 ## Abstract
 
+### release_request_received format
+
 As of now, the `btcDestinationAddress` field logged by the `release_request_received` event is a hash160 bytes format which is not user friendly. This RSKIP proposes updating the `btcDestinationAddress` field to the string format which is more user friendly.
 
+### New pegout_confirmed event
+
+Currently no event is being emitted when a pegout that is waiting for confirmation (in the `releaseTransactionSet` structure) has the required confirmations and is moved to waiting for signatures (to the `rskTxsWaitingForSignatures`).
+Right now, to check if a pegout has enough confirmation and that is moved to the `rskTxsWaitingForSignatures` structure, we need to get the state of the bridge and look for the pegout in `releaseTransactionSet` or `rskTxsWaitingForSignatures`, which is not that performant or convenient.
+
+We need to add a new event to the Bridge that will be emitted when the Bridge moves a pegout to the `rskTxsWaitingForSignatures`.
+I propose this event is called `pegout_confirmed`.
+
+### add_signature event logging order
+
+At the moment the `add_signature` event is being emitted before actually adding the signature.
```

Comment:
I think this could fit better into the Motivation or Specification sections. We could simply add a summary here explaining the purpose of this RSKIP (to improve UX by emitting meaningful events)